.github/workflows/e2e-test-dispatch-workflow.yaml

```diff
@@ -0,0 +1,16 @@
+name: ARC-REUSABLE-WORKFLOW
+on:
+  workflow_dispatch:
+    inputs:
+      date_time:
+        description: 'Datetime for runner name uniqueness'
```

Comment:
It'll be helpful to add either a comment or preferably include an example of the Date time format in the description